llvm/test/Transforms/LoopVectorize/AArch64/prefer-fixed-if-equal-to-scalable.ll

```diff
@@ -1,14 +1,21 @@
-; RUN: opt -S < %s -passes=loop-vectorize -force-target-instruction-cost=1 | FileCheck %s
+; RUN: opt -S < %s -passes=loop-vectorize -mcpu=generic | FileCheck %s --check-prefix=CHECK-GENERIC
```

Comment:
nit: Could you check the name of the CHECK line to something like CHECK-PREF-FIXED because when I first read this I thought it meant "prefixed". :)